opencog/rule-engine/InferenceSCM.cc

```diff
@@ -79,16 +80,17 @@ void InferenceSCM::init(void)
 #endif
 }
 
-Handle InferenceSCM::do_forward_chaining(Handle h)
+Handle InferenceSCM::do_forward_chaining(Handle h, const string& conf_path)
```

Comment:
My feeling is that after we've replaced JSON by Scheme, we can just load the configuration in the AtomSpace, before running cog-fc, and then we can remove the config path from cog-fc. Till then though, in order to get both cog-fc work on different rule systems (PLN, R2L, etc), both the atomspace repo and opencog repo. I don't see another solution.

Or maybe setting the config path in the atomspace conf file, but I don't think there is one on the atomspace repo.